pkg/KubeArmorMacro/PROJECT

```diff
@@ -0,0 +1,7 @@
+domain: accuknox.com
```

Comment:
the same thing.
please use kubearmor.com instead of accuknox.com in KubeArmorMacro too.